src/platform/packages/private/kbn-repo-packages/package-map.json

```diff
@@ -0,0 +1,4198 @@
+[
```

Comment:
Q: The linter wants this filename to be in snake case, but why is this file needed in 8.17 backport but not in `8.18` or `8.16`.